src/libs/Notification/PushNotification/index.native.ts

```diff
@@ -31,10 +31,15 @@ function pushNotificationEventCallback(eventType: EventType, notification: PushP
 
     // On Android, some notification payloads are sent as a JSON string rather than an object
     if (typeof payload === 'string') {
-        payload = JSON.parse(payload) as string;
+        try {
+            payload = JSON.parse(payload) as string;
```

Comment:
```suggestion
            payload = JSON.parse(payload) as JsonObject;
```

do same in other places